nussknacker-dist/src/universal/conf/users.conf

```diff
@@ -29,17 +29,17 @@ rules: [
   {
     role: "Reader"
     permissions: ["Read"]
-    categories: ["Default", "FraudDetection", "Recommendations"]
+    categories: ["Default", "FraudDetection", "Recommendations", "ServerRestApi", "Category1", "Category2"]
```

Comment:
We don't use them I think?

Reply:
We will do, when I move `develConf/application.conf` here - not done yet.